src/Common/tests/gtest_makeRegexpPatternFromGlobs.cpp

```diff
@@ -8,14 +8,29 @@ using namespace DB;
 
 TEST(Common, makeRegexpPatternFromGlobs)
 {
+
```

Comment:
Thanks for adding the new tests. To make this even more systematic and just as a suggestion: We could add "sub-cases" for "case 1" and "case 2" where 1) the left range width == the right range width, 2) then examples where the left range width > the right range width and 3) finally examples where the left range width < the right range width. Sub-case 2) can only be achieved using leading zeros on the left side, which means we might merge case 3 into cases 1 and 2.)